ReactWindows/ReactNative.Shared/UIManager/UIManagerModule.Constants.cs

```diff
@@ -76,6 +84,15 @@ public partial class UIManagerModule
             return constants;
         }
 
+        private static IDictionary<string, object> GetDefaultExportableEventTypes()
+        {
+            return new Map
+            {
+                { BUBBLING_EVENTS_KEY, GetBubblingEventTypeConstants() },
+                { DIRECT_EVENTS_KEY, GetDirectEventTypeConstants() },
+            };
```

Comment:
Suddenly we have to make sure we don't add all the view manager events here...It does make a difference, perf wise.

Reply:
This is only retrieved once - so it actually doesn't matter.

---
In reply to: [174848269](https://github.com/Microsoft/react-native-windows/pull/1717#discussion_r174848269) [](ancestors = 174848269)

Reply:
Yes - I suspect it should shave 100-200ms off the startup time, but testing is still in progress.